Microsoft.Research/Contracts/WindowsBase/System.Windows.Rect.cs

```diff
@@ -16,6 +16,19 @@
 using System.ComponentModel;
 using System.Diagnostics.Contracts;
 
+
+// 2015-03-36: tom-englert
+// Temporarily disabled checks for Contract.Requires(!this.IsEmpty);
```

Comment:
I doubt the checker will be ever capable verifying this particular case, and with this check in place code is unmaintainable, due to false positives in every line and even `Contract.Assume` does not help in many situations.

I will move this to a separate pull request and create an issue for that.